.prettierrc

```diff
@@ -1,6 +1,7 @@
 {
     "tabWidth": 4,
     "semi": false,
+    "useTabs": true,
```

Comment:
yeah, no![image](https://github.com/user-attachments/assets/86d40fa1-3d38-4729-bba9-36a27f000935)

Reply:
But in the configuration file of Vitepress the tabs are used

Reply:
But in the configuration file of Vitepress the tabs are used

Reply:
,,

Reply:
Something saltyaom did once is not for a whole project, there is prettier to format some pages, but we don't use tabs anywhere else

please revert this back

Reply:
🙏